File: _persist/temp/venda/CSharp/Terminal.cs
using MORM.CSharp.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("TERMINAL")]
    public class Terminal : CollectionItem
    {
        [Campo("ID_TERMINAL", CampoTipo.tfKey)]
        public int Id_Terminal { get; set; }
        [Campo("U_VERSION", CampoTipo.tfNul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.tfReq)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.tfReq)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("CD_TERMINAL", CampoTipo.tfReq)]
        public int Cd_Terminal { get; set; }
        [Campo("DS_TERMINAL", CampoTipo.tfReq)]
        public string Ds_Terminal { get; set; }
    }
}

File: _persist/temp/venda/Java/Terminal.java
package MORM.Java.Models;

import java.util.Date;
import MORM.Java.Classes;

@Tabela("TERMINAL")
public class Terminal extends CollectionItem
{
    @Campo("ID_TERMINAL", CampoTipo.tfKey)
    public Integer Id_Terminal;
    @Campo("U_VERSION", CampoTipo.tfNul)
    public String U_Version;
    @Campo("CD_OPERADOR", CampoTipo.tfReq)
    public Integer Cd_Operador;
    @Campo("DT_CADASTRO", CampoTipo.tfReq)
    public DateTime Dt_Cadastro;
    @Campo("CD_TERMINAL", CampoTipo.tfReq)
    public Integer Cd_Terminal;
    @Campo("DS_TERMINAL", CampoTipo.tfReq)
    public String Ds_Terminal;
}